Library/Homebrew/build.rb

```diff
@@ -100,6 +100,15 @@ def install
         ENV.prepend "LDFLAGS", "-L#{dep.opt_lib}" if dep.opt_lib.directory?
         ENV.prepend "CPPFLAGS", "-I#{dep.opt_include}" if dep.opt_include.directory?
       end
+
+      if OS.linux?
+        formula_deps.each do |dep|
+          ENV.append_path "LD_RUN_PATH", dep.opt_lib
+        end
+        unless ENV["LD_RUN_PATH"].empty?
+          ENV.prepend "LDFLAGS", "-Wl,-rpath,#{ENV["LD_RUN_PATH"]}"
```

Comment:
That's not ideal, but also won't break anything. That might happen if the tool's build system also uses a `-Wl,-rpath` option.